index.js

```diff
@@ -242,10 +242,12 @@ function stdin() {
  */
 
 function renderFile(path, rootPath) {
-  var re = /\.(?:pug|jade)$/;
+  var isPug = /\.(?:pug|jade)$/;
+  var isIgnored = /(\/_)|(^_)/;
```

Comment:
Currently your regex only looks for forward slashes (`/`), while MS-DOS (i.e. Windows) paths use backslashes (`\`).